libpius/signer.py

```diff
@@ -618,6 +619,9 @@ def sign_uid(self, key, index, level):
         print '  UID already signed'
         gpg.stdin.write('quit\n')
         return False
+      elif (PiusSigner.GPG_KEY_CONSIDERED in line):
+        debug("Got KEY_CONSIDERED")
+        continue
```

Comment:
I haven't looked closely enough to see why this was needed for you but not for me, but I've fixed it and normalized the style of that change from @civodul's patch.